### PR TITLE
SAI PTF 3: sairif

### PR DESCRIPTION
**Pull request by pkochmanska**

### CODE REVIEW

**Comment by ravi861** [comment]
@pkochmanska Can you please resolve the LGTM alerts?

File: ptf/sairif.py [hunk 1]
```diff
@@ -0,0 +1,7955 @@
...
+            self.client, self.route_lag0, next_hop_id=self.nhop3)
+
+        self.route_lag1 = sai_thrift_route_entry_t(
+            vr_id=self.default_vrf,
+            destination=sai_ipprefix(
+                '1234:5678:9abc:def0:1122:3344:5566:7788/128'))
+        sai_thrift_create_route_entry(
+            self.client, self.route_lag1, next_hop_id=self.nhop3)
+
+    def runTest(self):
```
[the remark this answers is not in the document]
Comment (pkochmanska) [reply ptf/sairif.py:112]: The assumption was to divide router interface tests into 3 classes - L3 interface, Sub-port and SVI and under each class have tests with common configuration. In my opinion I would leave it as it is now because it's more clear to understand the nature of these tests, but I could change it if you need smaller classes, but I'm not sure if it wouldn't just duplicate the config code.

**Review by richardyu-ms — APPROVED** [review]

File: ptf/sairif.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (richardyu-ms) [reply ptf/sairif.py:112]: try to resolve this comment from the PTF stcuture
https://github.com/p4lang/ptf/pull/168
